pkg/kube/controllers/versionedsecret/validating_webhook.go

```diff
@@ -65,17 +66,17 @@ type ValidationHandler struct {
 
 // NewValidationHandler returns a new ValidationHandler
 func NewValidationHandler(log *zap.SugaredLogger) admission.Handler {
-	validationLog := log.Named("secret-validator")
+	validationLog := log.Named("secret-validator").Desugar().WithOptions(zap.AddCallerSkip(-1)).Sugar()
```

Comment:
fancy

Reply:
That's needed to make the code locations accurate... maybe we can clean that up (understand why -1) when doing https://www.pivotaltracker.com/story/show/170841351